### ISSUE TITLE
[Feature Request] Folders on home screen

### ISSUE THREAD

**Issue by dallyger** [SimpleMobileTools/Simple-Launcher#33]
It would be nice to create folders on the fly by dragging icons on top of each other to declutter the home screen.

**Comment by tibbi** [comment]
duplicate of #20

**Comment by dallyger** [comment]
Hi @tibbi,

that's what I thought too at first.
But, to my understanding, the home screen is not the same as the app drawer or am I wrong?

I would either suggest reopening this ticket, or maybe change the title of #20 to a more generic "Folder Support" if a parallel implementation on both screens is possible and track the progress over there.

My bad, I could've explained my reasoning while opening the ticket directly to make my intention clearer.

**Comment by tibbi** [comment]
right, I plan adding them in home screen, but definitely not app drawer

**Comment by Saijin-Naib** [comment]
> right, I plan adding them in home screen, but definitely not app drawer

Any reason why not? They are incredibly useful on an app drawer, IMO.

Mine would be a nightmare without app drawer folders.
![Screenshot_20221218-000655.png](https://user-images.githubusercontent.com/19295950/208282440-08ce5e1d-2288-4253-b4b8-26bcecf18293.png)

**Comment by tibbi** [comment]
having this many icons is an extreme edge case, we focus on casual users
